fs/src/fs.cpp

```diff
@@ -245,6 +288,135 @@ void cleanup(char* buffer, int size, const FileHandle& handle)
     uv_fs_close(uv_default_loop(), &closeReq, handle.fileDescriptor, nullptr);
 }
 
+int fs_remove(lua_State* L)
+{
+    std::filesystem::remove(luaL_checkstring(L, 1));
```

Comment:
Does libuv really not have a function to remove files?

Reply:
It doesn't for some reason

Reply:
```c
int uv_fs_unlink(uv_loop_t* loop, uv_fs_t* req, const char* path, uv_fs_cb cb);
```?